.travis.yml

```diff
@@ -5,16 +5,13 @@ os:
 language: go
 
 go:
-    - 1.9
-
-install: true
```

Comment:
Note: `install: true` isn't technically useless. It disables the default `go` install command `go get -t ./...`. However, we might as well leave that in (we do a gx rw first so it should be a noop).

Reply:
@Stebalien from what I can see in the build output, install still does nothing even without this line, have a look at 580 of this build output:
https://travis-ci.org/libp2p/go-tcp-transport/jobs/303399619

Am I missing something, should I add back all the `install: true` I removed?

Reply:
Ah. It turns out that travis does some automagic (😡) here and disables the install step if there's a `Makefile`.

https://docs.travis-ci.com/user/languages/go/#Dependency-Management

Reply:
Regardless, it shouldn't really matter.

Reply:
From IRC
```
<kevina> stebalien: concerning "install: true" how about we make it "install: \n - make deps" instead of doing it in "before_install"?
<kevina> "make deps" is basically our version of "go get ./..." so its doing the same type of thing
<kevina> I found "install: true" confusing at first because I thought it meant to do the install and not the literal "true" command :)
<stebalien> kevina: we should absolutely be doing that.
```